src/products-view/Products.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import ProductDesktop from './ProductDesktop';
+import ProductMobile from './ProductMobile';
+import {useState, useEffect} from 'react'
+
+
+const ViewProducts = () => {
+
+    const [mobileText, setMoblieText] = useState(getWindow())
+    // const [isEmpty, setIsEmpty] = useState(true)
+
+    function getWindow () {
+        const {innerWidth} = window
+        if(innerWidth > 1280){
+            return false
+        }else{
+            return true
+        }
+    }
+
+    useEffect(() => {
+        //check for screen size
+        const switchView = () => {
+            setMoblieText(getWindow())
+        } 
+        window.addEventListener('resize', switchView)
+        return () => {
+            window.removeEventListener('resize', switchView)
+        }
+    },[])
```

Comment:
This is an optional suggestion because it doesn't break the code but rather shows your amazing coding skills. Great work handling the responsiveness of the website with this logic. I suggest converting this to a custom hook by creating a custom hook folder and exporting this logic there. Doing this would allow other developers to make use of the windowresize function and reduce code redundancy.